sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -69,16 +69,19 @@ statement
     | ALTER DATABASE identifier SET DBPROPERTIES tablePropertyList     #setDatabaseProperties
     | DROP DATABASE (IF EXISTS)? identifier (RESTRICT | CASCADE)?      #dropDatabase
     | createTableHeader ('(' colTypeList ')')? tableProvider
-        (OPTIONS tablePropertyList)?
+        (OPTIONS options=tablePropertyList)?
         (PARTITIONED BY partitionColumnNames=identifierList)?
-        bucketSpec? (AS? query)?                                       #createTableUsing
+        bucketSpec?
+        (TBLPROPERTIES properties=tablePropertyList)?
+        (COMMENT comment=STRING)?
```

Comment:
Do we need to keep the same order? For example, moving `(COMMENT comment=STRING)?` before `(PARTITIONED BY partitionColumnNames=identifierList)?`?